test/parallel/test-preload.js

```diff
@@ -62,6 +63,23 @@ childProcess.exec(
   }
 );
 
+// Test that preload can be used with --frozen-intrinsics
+childProcess.exec(
+  `"${nodeBinary}" --frozen-intrinsics ${preloadOption([fixtureE])}-e "console.log('hello');"`,
+  function(err, stdout) {
+    assert.ifError(err);
+    assert.strictEqual(stdout, 'hello\n');
```

Comment:
Could we maybe print something that actually checks that the polyfill has been loaded (and not just e.g. silently failed)?

Reply:
we could, but then we should probably name it something that will never be used by the language... maybe [Array.prototype.flatten](https://developers.google.com/web/updates/2018/03/smooshgate) ?

Reply:
Add something to Object.prototype - that will never be done by the language.

Reply:
done